src/Components/Forms/DiagnosisAndRecommendations/DecisionOnRelease/index.js

```diff
@@ -31,8 +32,30 @@ const RecommendationsOnRelease = () => {
   useEffect(() => {
     register({ name: 'decision' });
     register({ name: 'evacuationWay' });
+    const { items } = questionnaireResponse;
+    if (items) {
+      items.forEach((item) => {
+        if (item.answer) {
+          switch (item.text) {
```

Comment:
the switch needs to be on the linkId, the text can be changed. it's better to put the text in a comment then the code will be more readable